org.freedesktop.dabrain34.squared.json

```diff
@@ -0,0 +1,46 @@
+{
+  "app-id": "org.freedesktop.dabrain34.squared",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "45",
+  "sdk": "org.gnome.Sdk",
+  "command": "squared",
+  "finish-args": [
+    "--socket=fallback-x11",
+    "--socket=puseaudio",
+    "--device=dri",
+    "--share=ipc",
+    "--socket=x11",
+    "--persist=.local/share/godot/app_userdata/squared"
+  ],
+  "modules": [
+    {
+      "name": "squared",
+      "buildsystem": "simple",
+      "sources": [
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/f52198b93be7f495a85c9f8e70c34c5e/squared_x86_64"
+        },
+        {
+          "type": "file",
+          "url": "https://gitlab.freedesktop.org/dabrain34/squared/uploads/5fe7bf6de6f1d7a3c782822fd6b63732/data.pck"
+        }
+      ],
+      "build-commands": [
+        "install -Dm755 squared_x86_64 $FLATPAK_DEST/bin/squared",
+        "install -Dm555 data.pck $FLATPAK_DEST/bin/data.pck"
+      ]
+    },
+    {
+      "name": "squared2",
```

Comment:
why not put everything into the same module?

Reply:
That's what I wanted initially but I dont find how to implement a *postbuild* in the meson module.

Reply:
`post-install`.